assets/pages/support-landing/supportLanding.jsx

```diff
@@ -3,30 +3,50 @@
 // ----- Imports ----- //
 
 import React from 'react';
+import { Provider } from 'react-redux';
 
 import SimpleHeader from 'components/headers/simpleHeader/simpleHeader';
 import Footer from 'components/footer/footer';
 import CirclesIntroduction from 'components/circlesIntroduction/circlesIntroduction';
 import ThreeSubscriptions from 'components/threeSubscriptions/threeSubscriptions';
 import WhySupport from 'components/whySupport/whySupport';
 import ReadyToSupport from 'components/readyToSupport/readyToSupport';
-import PatronsEvents from 'components/patronsEvents/patronsEvents';
 
+import { init as pageInit } from 'helpers/page/page';
 import { renderPage } from 'helpers/render';
 
+import type { CommonState } from 'helpers/page/page';
+
+import PatronsEvents from './components/patronsEvents';
+
+
+// ----- Types ----- //
+
+export type State = {
+  common: CommonState,
+  page: {||},
+};
+
+
+// ----- Redux Store ----- //
+
+const store = pageInit({});
+
 
 // ----- Render ----- //
 
 const content = (
-  <div>
-    <SimpleHeader />
-    <CirclesIntroduction />
-    <ThreeSubscriptions />
-    <WhySupport />
-    <ReadyToSupport ctaUrl="#" />
-    <PatronsEvents />
-    <Footer />
-  </div>
+  <Provider store={store}>
```

Comment:
So does this just mean that `store` becomes accessible to all the components within?

Reply:
[Yup](https://github.com/reactjs/react-redux/blob/master/docs/api.md#provider-store).